arrow-flight/examples/flight_sql_server.rs

```diff
@@ -725,9 +732,8 @@ impl FlightSqlService for FlightSqlServiceImpl {
 /// This example shows how to run a FlightSql server
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let addr = "0.0.0.0:50051".parse()?;
-
-    let svc = FlightServiceServer::new(FlightSqlServiceImpl {});
+    let addr_str = "0.0.0.0:50051";
+    let addr = addr_str.parse()?;
```

Comment:
```suggestion
    let addr = "0.0.0.0:50051".parse()?;
    let svc = FlightServiceServer::new(FlightSqlServiceImpl {});
```

Minor nit, but we can revert to what it was previously (and down below too)